styles/styles.css

```diff
@@ -499,3 +499,43 @@ a.navigation::after {
 a.navigation:hover::after {
   transform: scaleX(1);
 }
+
```

Comment:
As discussed, I think we should use the templating mechanism for styles for this - IIRC, there already is a cocktails.css